doc/keyboard-layout-customization.md

```diff
@@ -0,0 +1,33 @@
+# How to customize your keyboard layout with System76 EC
+
+## Dependencies
+* Install [sdcc](http://sdcc.sourceforge.net/): `sudo apt install sdcc`
+* Install rust nightly
+  - Install [rustup](https://rustup.rs/)
+  - Reboot
+  - Run `rustup install nightly`)
+  
+## Adding your layout
+* In `src/board/system76/{your-model}/keymap/`, copy `default.c` and rename it. There are two examples to reference in the `lemp9` directory: `jeremy.c` and `levi.c`.
+* In `src/common/include/common/keymap.h` you will find a list of the key definitions.h
+* You will notice two sets of keys in these layout files. The top one is the standard mapping. The bottom one is the Fn layer, meaning it is active when the Fn key is being held. If you look at the Fn layer in the `levi.c` layout, you will see that there are arrow keys at WASD, media and volume keys on the bottom row, etc. 
+* I find it easiest to change one key at a time, referencing the key code I am deleting for keeping myself positioned correctly in the list of keycodes.
```

Comment:
```suggestion
* Hint: To avoid losing your place change one key at a time, referencing the key code you are deleting to keep yourself positioned correctly in the list of keycodes.
```